lib/vim_state.ahk

```diff
@@ -189,7 +200,7 @@
   }
 
   SetStatusCheck(){
-    check := this.StatusCheckObj
+    check := ObjBindMethod(this, "StatusCheck")
```

Comment:
Is there any merit to get ObjBindMethod directly?

Reply:
Nothing other than that it is much cleaner. Previously we had this:

```
this.StatusCheckObj := ObjBindMethod(this, "StatusCheck")
check := this.StatusCheckObj
```

I don't see why we needed it, although it is possible I missed another used of `this.StatusCheckObj`. I did search the rest of the project for it before removing it, however.

Reply:
I don't remember exactly, but this ObjBindMethod works so annoying.
I tested a lot of way and there could be a problem when ObjBindMethod is made on the fly.

Reply:
Ah, ok. In that case I'll revert it just in case, although it did seem fine when I used it